frontend/shared/containers/status/index.js

```diff
@@ -1,10 +1,114 @@
+import _ from "lodash"
 import React from "react"
+import { bindActionCreators } from "redux"
+import { connect } from "react-redux"
+import { loadStatus } from "../../modules/status"
+import { Popover, PopoverBody } from 'reactstrap';
 
 import "./style.css"
 
-export default () => (
-    <div className="status">
-        <span className="light success" />
-        Online
-    </div>
-)
+class Status extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = { loading: true }
+    }
+
+    componentDidMount() {
+        this.props.loadStatus()
+        this.interval = setInterval(this.props.loadStatus, 5000);
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (!nextProps.status && this.props.status) {
+            this.props.loadStatus()
+        }
+    }
```

Comment:
Is this meant to trigger the initial load?

Reply:
If so you can just call it in `constructor`

Reply:
DidMount does it, this is scheme I made all over the place that is originally meant for situations in which some operation triggers removing specific state from redux so when component that needs it is updated, it's refetched from backend. In this case it indeed doesn't need to be there. Will remove.